src/mixins/filing-template-mixin.ts

```diff
@@ -602,6 +602,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // restore Dissolution data
     this.setBusinessAddress(draftFiling.dissolution.custodialOffice)
     this.setDissolutionType(draftFiling.dissolution.dissolutionType)
+    // setting party
+    // my be only for SP GP : Need to discuss
+    this.setOrgPersonList(draftFiling.dissolution.parties)
```

Comment:
Now i am setting parties to orgperson list, we can discuss, what will be a better approach

Reply:
Is it possible to save a firm dissolution draft, and if so, what happens when you reload the draft?

Reply:
Obsolete code. Resolving.